msteams-platform/graph-api/activity-feed/test-activity-feed-json-file.md

```diff
@@ -0,0 +1,706 @@
+---
+title: Testing activity feed notifications in Teams
+description: JSON  test file resource-specific consent in Teams using Postman
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: How-to
+keywords: teams authorization OAuth SSO AAD rsc Postman Graph json
+---
+
+# Test RSC Postman collection JSON
+
+>[!NOTE]
+> Depending on the user you want to send notification **from** and the **target user/recipient** you may have to update those parameter in the payload.
+
+```json
+{
+	"info": {
+		"_postman_id": "cc85275b-fa1b-48c2-ac00-8bafe945146c",
+		"name": "Activity Feed Demonstrations",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "User Context",
+			"item": [
+				{
+					"name": "Authenticate",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "2e6527fc-b97d-4ac2-a412-3aad93d338ee",
+								"exec": [
+									"pm.test(\"Status Code 200\", () => {",
+									"    // Need to validate the request succeeded. ",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"// Set the user_context access token",
+									"const { access_token } = pm.response.json();",
+									"pm.environment.set(\"graph_access_token\", access_token);"
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "16050487-fbb3-4a3b-b471-cb6ebc746579",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"name": "Content-Type",
+								"type": "text",
+								"value": "application/x-www-form-urlencoded"
+							}
+						],
+						"body": {
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "grant_type",
+									"value": "password",
+									"type": "text"
+								},
+								{
+									"key": "username",
+									"value": "AlexW@M365x347208.OnMicrosoft.com",
+									"type": "text"
+								},
+								{
+									"key": "password",
+									"value": "07Apples!",
+									"type": "text"
+								},
+								{
+									"key": "client_id",
+									"value": "f97677f9-8607-476e-9759-8c45b2832b59",
+									"type": "text"
+								},
+								{
+									"key": "client_secret",
+									"value": "3RQa7TYXTU9ezMEu/O4x8pSNtkpM8tpW8m3foVjPVSc=",
+									"type": "text"
+								},
+								{
+									"key": "scope",
+									"value": "https://canary.graph.microsoft.com/.default",
```

Comment:
Can you please replace all instances of 'canary.graph.microsoft.com' to 'graph.microsoft.com'? We want to share the 'graph' endpoint instead of the 'canary graph' endpoint.